app/lib/utils/btcpayserver.js

```diff
@@ -0,0 +1,19 @@
+import get from 'lodash.get'
+
+const parseConnectionString = value => {
+  let config = {}
+  try {
+    config = JSON.parse(value)
+  } catch (e) {
+    return new Error('Invalid connection string')
+  }
+  const configs = get(config, 'configurations', [])
```

Comment:
Ideally configs should be named differently. like `allConfigs` to avoid very close looking name with `config`